### DISCUSSION TITLE
cannot undo pdm --pep582

### DISCUSSION THREAD

**Discussion by pbaetens** [pdm-project/pdm#2708] [Q&A] [answered]
On windows I did the following:

pdm --pep582

from then on, a __pypackages__ folder is made inside the project. The .venv is no longer created.

how to undo this?
I checked the PATH variable, but can't find anything suspicious.

https://pdm-project.org/latest/usage/pep582/#enable-pep-582-globally

**Comment by pawamoy** [comment, marked as answer]
You can configure the `python.use_venv` setting, setting it to true: `pdm config python.use_venv true`. Add `-g` for global configuration.
